UtahModules/Package.swift

```diff
@@ -47,7 +47,13 @@ let package = Package(
         .target(
             name: "UtahProfile",
             dependencies: [
-                .target(name: "UtahSharedContext")
+                .target(name: "UtahSharedContext"),
+                .product(name: "FHIR", package: "CardinalKit"),
+                .product(name: "Questionnaires", package: "CardinalKit"),
+                .product(name: "Scheduler", package: "CardinalKit"),
```

Comment:
I dont think you use these in the profile page, if you don't can you please take them out?